backend/src/main/scala/cromwell/backend/BackendWorkflowInitializationActor.scala

```diff
@@ -27,6 +32,60 @@ object BackendWorkflowInitializationActor {
   */
 trait BackendWorkflowInitializationActor extends BackendWorkflowLifecycleActor with ActorLogging {
 
+  /**
+    * Answers the question "does this expression evaluate to a type which matches the predicate".
```

Comment:
There's an `wdlExpression .evaluateType` method, looking at this it might be useful (or not)